lib/modules/manager/nuget/readme.md

```diff
@@ -1,5 +1,21 @@
-The `nuget` configuration object is used to control settings for the NuGet package manager.
-The NuGet package manager supports a SDK-style `.csproj`/`.fsproj`/`.vbproj`/`.props`/`.targets` format, as described [here](https://natemcmaster.com/blog/2017/03/09/vs2015-to-vs2017-upgrade/).
-This means that .NET Core projects are all supported but any .NET Framework projects need to be updated to the new `.csproj`/`.fsproj`/`.vbproj`/`.props`/`.targets` format in order to be detected and supported by Renovate.
+You can use the `nuget` configuration object to control the settings for the NuGet package manager.
+
+The NuGet package manager supports these SDK-style files and formats:
+
+- `.csproj`
+- `.fsproj`
+- `.vbproj`
+- `.props`
+- `.targets`
+
+.NET Core projects are supported by default.
+
+For Renovate to work with .NET Framework projects, you need to update these files so they match the new SDK-style format:
+
+- `.csproj`
+- `.fsproj`
+- `.vbproj`
+- `.props`
+- `.targets`
 
 The NuGet manager also supports `global.json` and `dotnet-tools.json` SDK files.
```

Comment:
Do we support `global.json` and `dotnet-tools.json` anymore?

Reply:
yes we do support them

https://github.com/renovatebot/renovate/blob/dca3418bbdbf1024ac6807e591562b4042bc846d/lib/modules/manager/nuget/index.ts#L10-L15